conda/gateways/connection/session.py

```diff
@@ -91,6 +94,7 @@ def __init__(self):
             self.mount("https://", http_adapter)
             self.mount("ftp://", FTPAdapter())
             self.mount("s3://", S3Adapter())
+            self.mount("rsa://", SecureIDAdapter())
```

Comment:
Is `rsa://` a documented/standardized URL scheme component for this server?

Reply:
Unfortunately not. I did some basic research on the matter using s3:// as an example. Couldn't figure out how to connect to amazon using that prefix (while using requests). So I figured I might be allowed to do this for servers protected by RSASecurID.

I realize it's pretty hacky! Suggestions?

Reply:
Perhaps I should try to figure out if _this_ server is protected by RSA, do all the things...

Reply:
I think adding support for 2FA and other (HTTP-based) authentication mechanisms to conda is a good idea. But I want to avoid starting (or continuing, I guess) the pattern of requiring to create a custom adapter for every authn scheme, especially for schemes like this that appear to be just slightly modified HTTP requests.

I'm going to bump this out of the 4.10.2 release, so we can maybe discuss some other, more generic ways of handling HTTP-based authn.

Reply:
Understood. I can try to dig around Conda's internals and see if I can come up with something on my own as well. I'll gladly post another PR if/when that occurs.

Cheers!